extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -145,7 +149,7 @@ public static Table readTable(
      * @see ParquetFlatPartitionedLayout
      */
     public static Table readTable(@NotNull final File sourceFile) {
-        return readTableInternal(sourceFile, ParquetInstructions.EMPTY);
+        return readTableInternal(sourceFile.toURI(), ParquetInstructions.EMPTY);
```

Comment:
We shouldn't have to go from File -> URI -> Path; ideally, there would still be file-specific (or Path-specific) method.